mongodb_migrations/cli.py

```diff
@@ -41,32 +41,59 @@ def run(self):
             print("No previous migrations found")
 
         sys.path.insert(0, self.config.mongo_migrations_path)
-        self._domigrate()
+        {
+            Execution.MIGRATE: self._domigrate,
+            Execution.DOWNGRADE: self._dorollback
+        }[self.config.execution]()
 
     def _domigrate(self):
         for migration_datetime in sorted(self.migrations.keys()):
             if not self.database_migration_names or migration_datetime > self.database_migration_names[0]:
-                print("Trying to migrate version: %s" % self.migrations[migration_datetime])
+                print("Trying to upgrade version: %s" % self.migrations[migration_datetime])
                 try:
                     module = __import__(self.migrations[migration_datetime])
                     migration_object = module.Migration(host=self.config.mongo_host,
                                                         port=self.config.mongo_port,
                                                         database=self.config.mongo_database)
                     migration_object.upgrade()
                 except Exception as e:
-                    print("Failed to migrate version: %s" % self.migrations[migration_datetime])
-                    print("%s" % e.message)
+                    print("Failed to upgrade version: %s" % self.migrations[migration_datetime])
+                    print(e.__class__)
```

Comment:
why we need `e.__class__`?

Reply:
what's more, I'll bring `logging` in, so that we don't need to use print all the time, but this is not a part of this PR.

Reply:
I had to put class because message was null most of the time.. I thought printing class would give a better insight